package.json

```diff
@@ -28,6 +28,7 @@
         "publish:alpha": "pnpm -r publish --tag alpha"
     },
     "devDependencies": {
+        "@types/chrome": "^0.0.195",
```

Comment:
This dep should probably be specific to the extension package(s) right?

Reply:
Happy to fix in a follow-up PR.

Reply:
Cool, thanks!